sql/src/main/java/io/crate/execution/dsl/projection/WindowAggProjection.java

```diff
@@ -71,6 +77,13 @@ public WindowAggProjection(StreamInput in) throws IOException {
             List<Symbol> inputs = Symbols.listFromStream(in);
             functionsWithInputs.put(function, inputs);
         }
+        if (in.readBoolean()) {
+            int orderByIdexesLength = in.readVInt();
+            orderByIndexes = new int[orderByIdexesLength];
+            for (int i = 0; i < orderByIdexesLength; i++) {
+                orderByIndexes[i] = in.readVInt();
```

Comment:
Could we use `readVIntArray` ? Also - would it make sense to have an empty array for "no order by" instead of nullable?

Reply:
I'd rather not carry an empty array around if we don't have to (especially as the OrderBy is nullable already on the WindowDefintion, and generally a missing order makes sense in the context of windows). Unless you have a strong opinion on this I'd like to keep it like this.